fec.config.js

```diff
@@ -13,4 +16,22 @@ module.exports = {
    * Add additional webpack plugins
    */
   plugins: [],
-};
\ No newline at end of file
+  moduleFederation: {
+    exposes: {
+      './RootApp': path.resolve(__dirname, './src/AppEntry.tsx'),
+      './DashboardWidget': path.resolve(
+        __dirname,
+        './src/components/LearningResourcesWidget/LearningResourcesWidget'
+      ),
+    },
+    exclude: ['react-router-dom'],
+    shared: [
+      {
+        'react-router-dom': {
```

Comment:
There is no routing in learning resources.

Reply:
The widget has a link used in there https://github.com/RedHatInsights/learning-resources/blob/master/src/components/LearningResourcesWidget/LearningResourcesWidget.tsx#L8